src/patternfly/components/InputGroup/examples/input-group-example.hbs

```diff
@@ -1,45 +1,45 @@
 {{#> input-group }}
-  {{#> button btnClass="pf-m-secondary"}} Button {{/button}}
-  {{#> form-controls controlType="textarea" name="textarea1" id="textarea1" }}
+  {{#> button btnClass="pf-m-secondary" id="textAreaButton1"}} Button {{/button}}
+  {{#> form-controls controlType="textarea" name="textarea1" id="textarea1" aria-label="textarea with buttons" aria-describedby="textAreaButton1" }}
 {{/form-controls}}
     {{#> button btnClass="pf-m-tertiary"}} Button {{/button}}
 {{/input-group}}
 <br>
 <br>
 {{#> input-group }}
-  {{#> form-controls controlType="textarea" name="textarea2" id="textarea2" }}
+  {{#> form-controls controlType="textarea" name="textarea2" id="textarea2" aria-label="textarea with button" aria-describedby="textAreaButton2" }}
 {{/form-controls}}
-    {{#> button btnClass="pf-m-tertiary"}} Button {{/button}}
+    {{#> button btnClass="pf-m-tertiary" id="textAreaButton2"}} Button {{/button}}
 {{/input-group}}
 <br>
 <br>
 {{#> input-group }}
-  {{#> dropdown id="dropdown-example-collapsed"}}
+  {{#> dropdown }}
```

Comment:
Why remove the `id`? The dropdown component assumes that attribute exists, so you have empty `id=""` on the button and `aria-labelledby=""` on the dropdown menu.

Reply:
ah, I was just getting rid of the duplicate ID's I'll add them back in.